example/package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "eslint \"**/*.{js,ts,tsx}\" --fix",
     "lint-ci": "eslint \"**/*.{js,ts,tsx}\" -f @jamesacarr/github-actions",
     "bundle-install": "bundle install",
-    "pods": "cd ios && bundle exec pod install && rm -rf .xcode.env.local",
+    "pods": "cd ios && bundle exec pod install",
```

Comment:
btw: without this change i can't run the example app. I am pretty sure that deleting the .xcode.env.local is wrong.
Will remove this change in a moment though